app/controllers/lessons_controller.rb

```diff
@@ -13,9 +13,9 @@ def new
   end
 
   def create
-    @lesson = current_user.lessons.new(lesson_params)
+    @lesson = Lesson.new(lesson_params)
     if @lesson.save
-      redirect_to @lesson
+      render :show
     else
       render :new, status: :unprocessable_entity
```

Comment:
We do not need to render show here. We need to return an errors with validations or anything else.